document_rtf_index/README.rst

```diff
@@ -0,0 +1,37 @@
+Index rtf documents
```

Comment:
update this to the current template of the oca readme. Main difference is that all links should be https now